src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,145 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+
+type Props = {};
+
+const GENDER_FEMALE = 'f';
+const GENDER_MALE = 'm';
+const tableColumnNames = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+const NOT_SET_VALUE = '-';
+
+export const PeopleList: React.FC<Props> = () => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Better to have `false` by default

Reply:
Not fixed

Reply:
I thought tests won't pass, cuz when it's loading I can see like for half a second an empty table of people, so I set true by default to prevent such display